examples/java/src/main/java/org/apache/beam/examples/MinimalWordCount.java

```diff
@@ -63,13 +62,19 @@ public static void main(String[] args) {
     // Create a PipelineOptions object. This object lets us set various execution
     // options for our pipeline, such as the associated Cloud Platform project and the location
     // in Google Cloud Storage to stage files.
-    DataflowPipelineOptions options = PipelineOptionsFactory.create()
-        .as(DataflowPipelineOptions.class);
-    options.setRunner(BlockingDataflowPipelineRunner.class);
-    // CHANGE 1/3: Your project ID is required in order to run your pipeline on the Google Cloud.
-    options.setProject("SET_YOUR_PROJECT_ID_HERE");
-    // CHANGE 2/3: Your Google Cloud Storage path is required for staging local files.
-    options.setTempLocation("gs://SET_YOUR_BUCKET_NAME_HERE/AND_TEMP_DIRECTORY");
+    PipelineOptions options = PipelineOptionsFactory.create();
+
+    // In order to run your pipeline on the Google Cloud, you need to make following
+    // runner specific changes:
+
+    // CHANGE 1/4: Select a Beam runner, such as BlockingDataflowPipelineRunner,
```

Comment:
Change 1/X: Select a Beam runner, such as Blocking..Runner or FlinkPipelineRunner.
Change 2/X: Specify runner-required options. For Blocking..Runner, set project and temp location as follows. For XXXRunner, ...